test/test_version.py

```diff
@@ -0,0 +1,44 @@
+import os
+import re
+import unittest
+from puma.utils import PROJECT_ROOT
+from puma.version import version
```

Comment:
maybe use import as to give this value a better name

Reply:
Good one, changed it to setup_version